deprecations/migration-guide-get-reservations.md

```diff
@@ -0,0 +1,318 @@
+# Migration Guide: Get all reservations
+
+This guide is intended to assist API users to migrate away from the obsolete API Operation [Get all reservations (ver 2017-04-12)] which is long deprecated and will be permanently discontinued.
+
+> **Ensuring functional parity**
+> 
+> We've made efforts to ensure parity of functionality between the old and the new endpoint. However, it is possible we have not captured all of them. If you know of any functionality missing on the new endpoint that keeps you reliant on the old endpoint, please share your use case in this [short survey](https://mews.typeform.com/to/izhRLSFg).
+>
+> Additionally, if you would like to consult our Technical Partnerships team about your migration plans, you can contact them at <partnersuccess@mews.com>.
+
+> **Let us know about your timeline**
+> 
+> Do you know your migration timeline? Stuck on the original reservations endpoint due to missing functionality? Let us know on this [short survey](https://mews.typeform.com/to/izhRLSFg) so we can address it!
+>
+> Additionally, if you would like to consult our Technical Partnerships team about your migration plans, you can contact them at <partnersuccess@mews.com>.
+
+## Contents
+
+- [Get all reservations (ver 2023-06-06)](#get-all-reservations-ver-2023-06-06)
+- [Migration steps](#migration-steps)
+- [Request parameters](#request-parameters)
+- [Time filters](#time-filters)
+- [Extents removal](#extents-removal)
+- [Replacement operations for extents](#replacement-operations-for-extents)
```

Comment:
This is a sub-heading inside 'Extents removal' so I don't think it should be in the main Contents.
```suggestion
```